src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const mainImage = document.querySelector('.gallery__large-img');
+const thumbs = document.querySelector('#thumbs');
+
+thumbs.addEventListener('click', (mouseEvent) => {
+  const item = mouseEvent.target.closest('.gallery__thumb');
+
+  if (!item || !thumbs.contains(item)) {
+    return;
+  }
+  mouseEvent.preventDefault();
```

Comment:
It's more understandable if you write `.preventDefault()` right At the beginning of your code inside the listener